### ISSUE TITLE
2020 Berlin

### ISSUE THREAD

**Issue by blixtra** [cncf/kubernetes-community-days#177]
### Organizers 1: Chris Kuehl, Kinvolk, CEO, chris@kinvolk.io + Lexi Nadolski, Kinvolk, Head of Events, lexi@kinvolk.io
Chris is a CNCF Ambassador and long-time open source contributor.
Lexi heads the events team at Kinvolk and coordinates volunteer and event diversity efforts.
### Organizer 3: Tine Rehbein, SysEleven, Teamlead Marketing, Tine tine@syseleven.de

We'd like to propose an event in Berlin the 2nd half of May. This is a perfect time to hold an event in Berlin, with great weather and before the holidays kick in. Berlin also offers a wealth of unique venues and one of the most active tech scenes in Europe.

### Experience

We at Kinvolk already organize several successful open source events. For example, next week we have the 3rd iteration of [All Systems Go!](https://all-systems-go.io/), the conference for foundational user-space Linux technologies. In November we'll be putting on the 2nd [Cloud Native Rejekts](https://cloud-native.rejekts.io/) conference, an event that offers a stage to some of the wonderful, but rejected, KubeCon proposals. We also organize the more than 1000 member strong [CNCF Berlin Meetup](https://www.meetup.com/Cloud-Native-Computing-Berlin/).

Kinvolk focuses on putting on no-frills, engineer-oriented events and always looks forward to collaborating with others. This seems to be fully inline with what the CNCF is striving to do with KCD.

Loodse has organized [Container Days](https://www.containerdays.io/) in Hamburg and [GoDays](https://www.godays.io/) in Berlin. They bring lots of experience to the effort around all aspects of organizing community events.

### Submission checklist:

*Please check each of these indicating that you have met these requirements:

* [x] All organizers signed off to the event guidelines and CNCF Code of Conduct.
* [x] Method to accept payments including sponsorships and payout expenses
* [x] One organizer is a CNCF Member employee, Ambassador, or CNCF Project Maintainer
* [x] Minimum of 3 organizers from 3 different organizations
* [x] Minimum of 3 sponsors
* [x] Each organizer needs to live in or just outside of the KCD location
* [x] No other Kubernetes Community Days event in the same city
* [x] Agree to use Eventbrite for registration _(reluctantly)_
* [x] Agree to use Sched for organizing the program page _(reluctantly)_

### Method of accepting payment
Stripe or bank transfer preferred.

**Comment by christinevblum** [comment]
@blixtra Please continue to update as you complete the items on the checklist. Once complete we'll sign off on your team creating the event site.

**Comment by blixtra** [comment]
Sure, we have a meeting with some folks on Thursday. Will update after that meeting.

**Comment by blixtra** [comment]
@christinevblum I think we have our ducks in a row now. Would love to get the go ahead for this.

**Comment by blixtra** [comment]
Ah, didn't see that in the checklist. That'll be easy to get together. Will get back to you.

**Comment by dankohn** [comment]
That's because it wasn't there yet when you opened the issue! Thanks for accommodating our growing pains.

**Comment by blixtra** [comment]
Questions: To be clear, do the 3 sponsors have to be outside the organizing companies?

**Comment by cpanato** [comment]
@blixtra can i help somehow?

**Comment by smourapina** [comment]
I would also be happy to help @blixtra, will be back in Berlin from mid-December onwards!

**Comment by KTan1226** [comment]
Good news @blixtra, you are approved to build your site.

**Comment by blixtra** [comment]
Great! :confetti_ball:  We'll start organizing the kick-off meeting.

**Comment by cpanato** [comment]
@blixtra let me know when you are planning that :)

**Comment by dankohn** [comment]
@KTan1226 Can you please create the email alias.

Organizers, could you please share your Slack IDs.

**Comment by KTan1226** [comment]
Will do @dankohn.

**Comment by KrissiWi** [comment]
Here come the slack ideas from our side:

Bill: UCMGHPTD4
Krissi: U1SKBQZ89
Carolin: UNCP6LC0L

**Comment by KTan1226** [comment]
@KrissiWi Thank you. I will add you all to the KCD organizers Slack channel.

**Comment by LexiNadolski** [comment]
Here are the slack IDs for Chris and I:

Chris: U09UZDF9B
Lexi: U4BV6TYQJ

**Comment by dankohn** [comment]
Sorry, we want your CNCF Slack "Display Name" not the ID.

**Comment by LexiNadolski** [comment]
For Chris and I, respectively:

chris
lexi_nadolski

**Comment by cpanato** [comment]
my slack id on Kubernetes is `cpanato`

**Comment by smourapina** [comment]
My slack user is: `smourapina`

**Comment by LexiNadolski** [comment]
My kubernetes display name is: lexi_at_kinvolk

**Comment by LexiNadolski** [comment]
For Chris: @ckuehl on the CNCF slack, @chrisk on the kuberntes slack

**Comment by KTan1226** [comment]
Thanks, all. I add you to the channel shortly.

**Comment by xmulligan** [comment]
My slack is bill_loodse

**Comment by dankohn** [comment]
Ready to create your site? Can we help?

**Comment by KrissiWi** [comment]
Hi Dan, we'll have a meeting next week to fix a date and venue and will create the site after that. We'll let you know in case we need support. Thanks for asking!

**Comment by LexiNadolski** [comment]
@KTan1226 @dankohn We had a meeting on Wednesday and we have a few questions we hope to address with you. 

 - What is the price per person for tickets? We have heard different numbers and wonder if it varies by location. This will help us in our financial projections. 
- Is CNCF underwriting these events or does that fall on the organizers? 
- Will CNCF host the website or does one of the organizers have to take ownership?

I also have some housekeeping items. Could you please add Neza@kinvolk.io to the email alias? Also, I am unsure who to reach out to regarding the CNCF slack channel. I have to be added, as does Neza. 

Thank you in advance!

**Comment by KTan1226** [comment]
Hi @LexiNadolski. For pricing guidelines, please visit: https://kubernetescommunitydays.org/organizing-registration/. 

Can you explain what you mean by underwriting the events? Each event is run by the local organizers.

Also, each organizer is responsible for creating their own website once the checklist is complete. Guidelines on how to build your site can be found here: https://kubernetescommunitydays.org/organizing-creating-website/.

I will add Neza to the email alias. Can you share their Slack username? I will add you both to the KCD- organizers channel. Let me know if you have any additional questions!

**Comment by dankohn** [comment]
@LexiNadolski thanks for the questions. I would add:

>What is the price per person for tickets? We have heard different numbers and wonder if it varies by location. This will help us in our financial projections.

You need to decide! As @KTan1226 mentioned, please see https://kubernetescommunitydays.org/organizing-registration/.

>Is CNCF underwriting these events or does that fall on the organizers?

No, the organizers are taking on the risk for running the events.

>Will CNCF host the website or does one of the organizers have to take ownership?

Note that you are responsible for the content, but we host the website. Lots of details at https://kubernetescommunitydays.org/organizing-creating-website/.

**Comment by LexiNadolski** [comment]
@KTan1226 @dankohn Thank you for all of this information. 

I am not on the CNCF slack so I cannot provide my username. Can you invite me or let me know what I should do to get on it? Thanks!

**Comment by dankohn** [comment]
It's free to create an account at slack.cncf.io.

**Comment by KTan1226** [comment]
Hi Berlin team! Just checking in to see how the website is coming along. Is there anything we can do to help?

**Comment by KrissiWi** [comment]
Hey:) We finally fixed venue + date and are currently setting up website including CFP, registration and sponsoring prospectus. The plan was to launch next week, we hope we can stick to the timeline

**Comment by tinerehbein** [comment]
@KTan1226 we are ready to go online https://deploy-preview-246--kubernetes-community-days.netlify.com/events/2020-berlin/

Pls have a look and approve the side

**Comment by dankohn** [comment]
You're missing a link to the sponsor prospectus on https://deploy-preview-246--kubernetes-community-days.netlify.com/events/2020-berlin/sponsor/. Also, don't link from here: https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

Rather than Partners, I would change the sponsor category to "Initial Sponsors".

The kerning on "Berlin" in the background logo is off.

We're asking for you not to use Sessionize for your CFP, but instead to use SM Apply, which we'll be making available very shortly. https://kubernetescommunitydays.org/organizing-faq/#why-is-cncf-mandating-the-use-of-specific-tools-like-eventbrite-for-registration-sched-for-the-schedule-and-sm-apply-for-the-cfp

**Comment by dankohn** [comment]
You also need to specify what languages talks can be in

**Comment by cpanato** [comment]
@dankohn updated the first and second comments.

for the other will defer to others to check

**Comment by dankohn** [comment]
Great. I would encourage you to add the PDF to the assets directory and link to it there rather than linking to Google Drive. The latter causes problems for some people based on their corporate Google Docs configurations. Prospectus looks great, BTW.

**Comment by cpanato** [comment]
@dankohn I tried but then i was not able to download that, do you have an example in how to make it work?

**Comment by tinerehbein** [comment]
@cpanato attached the picture with more space or you can use the berlin image
![KCD-Berlin-Logo-space](https://user-images.githubusercontent.com/41007230/74044438-5b77fb80-49cb-11ea-9df2-a46a0b550730.png)
![the-famous-television-tower-berlin-web](https://user-images.githubusercontent.com/41007230/74044511-78143380-49cb-11ea-88b4-ddce9a9a9645.png)

**Comment by cpanato** [comment]
@tinerehbein updated with the drawing image

**Comment by tinerehbein** [comment]
@cpanato hm, it does look the same. which size of the image do you need?

**Comment by cpanato** [comment]
@tinerehbein the one was sent as SVG before, dont know the exactly size (sent you a request to talk via Google hangout we can chat there)

**Comment by KrissiWi** [comment]
@dankohn : Based on the discussion: https://github.com/cncf/kubernetes-community-days/issues/172 we assumed that there was no final decision on the CFP tooling yet. We now have set up and paid Sessionize for both, KCD Berlin & KCD Munich.

What do you think is a good way to proceed?

**Comment by tinerehbein** [comment]
@dankohn updated all your comments.

**Comment by dankohn** [comment]
@cpanato here is how to embed your PDF directly on the site: https://github.com/cncf/kubernetes-community-days/pull/284/files

You can ping me on Slack if it's not clear.

**Comment by cpanato** [comment]
@dankohn done! thanks for the tip!

**Comment by KrissiWi** [comment]
@dankohn : Any advise on how to proceed with the CfP tooling? Once this is solved, we can go live and announce.

**Comment by KTan1226** [comment]
Hi @KrissiWi! Did you get my message via Slack? Let's connect on this.

**Comment by KrissiWi** [comment]
Hi @KTan1226, thanks for getting back. I am available tomorrow after 4 pm CEST and anytime on Wednesday but since we wanted to launch this week, the earlier the better:) Just send me an invite to kristin@loodse.com

**Comment by KTan1226** [comment]
Great, @KrissiWi. I sent over a calendar invite for us to chat. Let me know if the time doesn't work. You can reach me on Slack (@Kristi Tan) or kristi@cncf (dot) io. Thanks!

**Comment by KTan1226** [comment]
Hi @KrissiWi and Berlin team,

I've added you all as admins for the KCD Berlin CFP via SM Apply. Please let me know if you have any questions or have trouble gaining access. 

The CFP is now live and can be found here: https://kcdcfpsubmissions.smapply.io/prog/kcd_berlin/.

Thanks!

**Comment by KrissiWi** [comment]
@dankohn and @KTan1226 : It may not come as a surprise: Given the situation in Europe as it is, we have decided to postpone KCD Berlin with no new date scheduled yet. I think we all agree that we don't want to put anyone at risk at this moment.

As a next step, we'll update the website and communicate the postponment to all actors involved so far. We have decided to sync again in 4-6 weeks time when the situation hopefully gets a bit clearer again. 

Is there anything else that we forgot?

Thank you for the support so far & we'll keep everyone updated.

**Comment by dankohn** [comment]
Makes sense. Thanks for the update.

**Comment by KTan1226** [comment]
Thanks @KrissiWi for the update. Do you need help pausing your CFP? Or would you like to keep it open for now with no end date?

**Comment by xmulligan** [comment]
@KrissiWi We are relaunching the KCD program, including virtual events. We would love to have one in Berlin if you are still interested! You can read [the blog post here](https://www.cncf.io/blog/2021/03/01/relaunching-kubernetes-community-days-with-kcd-africa-bengaluru/). If you have any questions, please contact me at kcd@cncf.io or [book a meeting with me](https://calendly.com/billmulligan/1-1-with-bill-mulligan?month=2021-03).

**Comment by cpanato** [comment]
@KrissiWi @xmulligan count on me to help out for the berlin edition

**Comment by xmulligan** [comment]
Glad to hear @cpanato !

**Comment by LexiNadolski** [comment]
Hello All @KrissiWi @tinerehbein @cpanato Maybe we can get a call going to see if we can talk about this further. I will circulate an email and a doodle. Hope to chat soon :)

**Comment by KrissiWi** [comment]
Cool - looking forward to be back in touch after such a long time:/

**Comment by LexiNadolski** [comment]
@KrissiWi @tinerehbein @cpanato Here is the Doodle to coordinate. I will also send this by email. I am really looking forward to seeing you all soon! :D 

https://doodle.com/poll/u56u6u2bigxcgfd5?utm_source=poll&utm_medium=link

**Comment by LexiNadolski** [comment]
@xmulligan we are planning to revive this event to take place March 2022. Should we update this issue or make a new one? Also, should we hold off until we have a date? Thanks!
